### ISSUE TITLE
[vtm-renderer] implement style:shapes other than circle for spatialite

### ISSUE THREAD

**Issue by brentfraser** [geopaparazzi/geopaparazzi#600]
Currently selecting point style to anything other than circle (e.g. square, triangle, etc) does not change the rendered point style.

**Comment by moovida** [comment]
Good point. This is one of the things that is not supported in vtm right now. I will keep this issue as reference to remove the shape selection from the dialog if I am not able to change that.
Please continue signaling such things.
